### PR TITLE
[infra] Mark more tests as integration

### PR DESCRIPTION
**Pull request by jonathanmetzman**

### CODE REVIEW

**Comment by jonathanmetzman** [comment]
TBR. Shouldn't affect tests as they are run on CI.

File: infra/build_specified_commit_test.py [hunk 1]
```diff
@@ -48,8 +50,8 @@ def test_build_fuzzers_from_commit(self):
       host_src_dir = build_specified_commit.copy_src_from_docker(
           test_case.project_name, tmp_dir)
 
-      test_repo_manager = repo_manager.RepoManager(
-          test_case.git_url, host_src_dir, repo_name=test_case.oss_repo_name)
+      test_repo_manager = repo_manager.clone_and_get_manager(
```
Comment (inferno-chromium) [inline infra/build_specified_commit_test.py:53]: Is this change correct ? i cannot find clone_and_get_manager, please fix if needed in TBR.

**Review by inferno-chromium — COMMENTED** [review]
LGTM with a question.

File: infra/build_specified_commit_test.py (hunk 1 above)
Comment (jonathanmetzman) [reply infra/build_specified_commit_test.py:53]: It's not correct. Really odd that the linter didn't catch this.
On the bright side, it's a disabled test.

Comment (jonathanmetzman) [reply infra/build_specified_commit_test.py:53]: Fixed in https://github.com/google/oss-fuzz/pull/4871

**Pull request merged**